src/main/java/HTML1/Task2.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <style>
+
+        body {
+            background-color: PeachPuff;
+        }
+
+        .box {
+            border: 3px solid red;
+            background-color: black;
+            padding: 40px;
+            margin: 40px;
+            width: 500px;
+            height: 130px;
+        }
+
+        .box1 {
+            border: 3px solid red;
+            background-color: black;
+            padding: 40px;
+            margin: 40px;
+            width: 500px;
+            height: 170px;
+        }
+    </style>
+
+    <title>index.html</title>
+
+
+</head>
+
+<body>
+
+<div class="box" align="justify" style="color:orange;font-family: sans-serif;text-indent: 25px;"> Медленный человек
```

Comment:
Why did this styles not in `.box` selector?
It seems, they also should be there